packages/common-scripts/src/omnilock.ts

```diff
@@ -35,17 +35,39 @@ import {
   createFixedHexBytesCodec,
 } from "@ckb-lumos/codec/lib/blockchain";
 import { bytify, hexify } from "@ckb-lumos/codec/lib/bytes";
+import * as bitcoin from "./omnilock-bitcoin";
 const { ScriptValue } = values;
 
 export type OmnilockInfo = {
-  auth: {
-    flag: "ETHEREUM" | "SECP256K1_BLAKE160";
-    /**
-     * if auth flag is SECP256K1_BLAKE160, content is publicKeyToBlake160(secp256k1Pubkey)
-     * if auth flag is ETHEREUM, content is Ethereum address
-     */
-    content: BytesLike;
-  };
+  auth: OmnilockAuth;
+};
+
+export type OmnilockAuth = IdentityCkb | IdentityEthereum | IdentityBitcoin;
+
+export type IdentityCkb = {
+  flag: "SECP256K1_BLAKE160";
+  /**
+   * the blake160 hash of a secp256k1 public key
+   */
+  content: BytesLike;
+};
+export type IdentityEthereum = {
+  flag: "ETHEREUM";
+
+  /**
+   * an Ethereum address, aka the public key hash
+   */
+  content: BytesLike;
+};
+export type IdentityBitcoin = {
+  flag: "BITCOIN";
+  /**
+   * a Bitcoin address, such as
+   * `P2PKH(17VZNX1SN5NtKa8UQFxwQbFeFc3iqRYhem)`,
+   * `P2SH(3EktnHQD7RiAE6uzMj2ZifT9YgRrkSgzQX)`,
+   * `Bech32(bc1qw508d6qejxtdg4y5r3zarvary0c5xw7kv8f3t4)`
+   */
+  address: string;
```

Comment:
What confused me is that thare are 2 fields in `IdentityEthereum` and `IdentityBitcoin`
- flag
- content | address
- 
As mentioned at line 58, `content` in `IdentityEthereum` is also an address, so why not use the same field, `address`, or both use `content`

Reply:
> why not use the same field, address, or both use content

the content here means the public key hash. However, the public key hash is an alias of the address on Ethereum.

Maybe we can change it to 

```ts
type IdentityEthereum = { flag: "Ethereum" } & ({content: BytesLike} | {address: BytesLike})
```

This would be more clear

Reply:
I've unified to use `content` to make it easier to use